core/src/main/resources/org/apache/spark/ui/static/stagepage.js

```diff
@@ -897,7 +897,7 @@ $(document).ready(function () {
                     // Get the column
                     var para = $(this).attr('data-column');
                     if (para == "0") {
-                        var column = taskTableSelector.column(optionalColumns);
+                        var column = taskTableSelector.columns(optionalColumns);
```

Comment:
Could you rename the variable to something like `allColumns` to have more clarity?